Modules/Relationships/src/Relationships.Domain/Aggregates/Relationships/Relationship.cs

```diff
@@ -78,7 +78,7 @@ private static void EnsureTargetIsNotSelf(RelationshipTemplate relationshipTempl
 
     private static void EnsureNoOtherRelationshipToPeerExists(IdentityAddress target, IEnumerable<Relationship> existingRelationshipsToPeer)
     {
-        if (existingRelationshipsToPeer.Any(r => r.Status is RelationshipStatus.Active or RelationshipStatus.Pending or RelationshipStatus.Terminated))
+        if (existingRelationshipsToPeer.Any(r => r.Status is RelationshipStatus.Active or RelationshipStatus.Pending or RelationshipStatus.Terminated or RelationshipStatus.DeletionProposed))
```

Comment:
can't this be simplified to Status != Rejected?

Reply:
No. There are also "Revoked" and "ReadyForDeletion".